README.md

```diff
@@ -67,6 +67,8 @@ For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse
      * [OPTIONAL] **--cache-poll-timeout-msec=1** : Timeout (milisecond) of checking the blobfuse cache for any possible eviction and free up space. Default is 1 milisecond.
      * [OPTIONAL] **--max-eviction=0** : How many cached files shall be evicted at a time, in case multiple are expired. This is to conserve the CPU usage done for cache eviction. Remaining files will be evicted in the next turn. '0' means there is no upper limit and all expired files will get evicted, may result in 100% CPU usage.
      * [OPTIONAL] **--set-content-type=false** : While uploading blob set the 'content-type' property based on file extension. Disabled by default, to enable set value to 'true'.
+     * [OPTIONAL] **--ca-cert-file=/etc/ssl/certs/proxy.pem** : If external network is only available through a proxy server, this parameter should specify the proxy pem certificate otherwise blobfuse cannot connect to the storage account.
+     * [OPTIONAL] **--httpsproxy=http://10.1.22.4:8080/** : If external network is only available through a proxy server, this parameter should specify the proxy server along with the port which is 8080 unless some alternatives have been made.
```

Comment:
We do support connection to container over http as well. Do we need to take care of http-proxy seperatly ?

Reply:
yes, we do but when you set http_proxy and https_proxy, normally you should set them at the same URL including the schema part. When, using the proxy for connecting to a HTTPS website, the proxy works in TCP mode, and your TCP connection passes through the proxy carrying encrypted packets from your client to the target website.

Reply:
added http_proxy as well so that it will be used for http only connections. Done!